src/screens/settings.rs

```diff
@@ -0,0 +1,138 @@
+//! The settings screen accessible from the title screen.
+//! We can add all manner of settings and accessibility options here.
+//! For 3D, we'd also place the camera sensitivity and FOV here.
+
+use std::marker::PhantomData;
+
+use bevy::{audio::Volume, ecs::system::IntoObserverSystem, prelude::*, ui::Val::*};
+
+use crate::{audio::GlobalMusicVolumeScale, screens::Screen, theme::prelude::*};
+
+const MIN_VOLUME: f32 = 0.0;
+const MAX_VOLUME: f32 = 3.0;
+
+pub(super) fn plugin(app: &mut App) {
+    app.add_systems(OnEnter(Screen::Settings), spawn_settings_screen);
+    app.add_systems(
+        Update,
+        update_music_volume_label.run_if(in_state(Screen::Settings)),
+    );
+}
+
+#[derive(Component)]
+struct MusicVolumeLabel;
+
+fn spawn_settings_screen(mut commands: Commands) {
+    let music_settings = PercentageSettings {
```

Comment:
I'm using a struct here and not passing it as params because I think passing 4 weird params looks unreadable. Especially if you happen to forget whether `on_minus` or `on_plus` goes first.